auth/api/iam/api.go

```diff
@@ -297,10 +299,50 @@ func (r Wrapper) HandleAuthorizeRequest(ctx context.Context, request HandleAutho
 	// Workaround: deepmap codegen doesn't support dynamic query parameters.
 	//             See https://github.com/deepmap/oapi-codegen/issues/1129
 	httpRequest := ctx.Value(httpRequestContextKey).(*http.Request)
-	params := make(map[string]string)
+	params := make(map[string]interface{})
 	for key, value := range httpRequest.URL.Query() {
-		params[key] = value[0]
+		if len(value) == 1 {
```

Comment:
why the change? can parameters occur multiple times? Note that `createSession` casts everything to string again

Reply:
obsolete due to addition of `singleStringParam`